pos_sale_order_delivery_carrier/static/src/js/Popups/CreateOrderPopup.esm.js

```diff
@@ -0,0 +1,90 @@
+/** @odoo-module **/
+
+import CreateOrderPopup from "point_of_sale.CreateOrderPopup";
+import Registries from "point_of_sale.Registries";
+import framework from "web.framework";
+
+const {useState} = owl;
+
+const PosDeliveryCarrierCreateOrderPopup = (CreateOrderPopup) =>
+    class extends CreateOrderPopup {
+        // @override
+        setup() {
+            super.setup();
+            this.state = useState({addDeliveryCarrier: false});
+        }
+        get currentOrder() {
+            return this.env.pos.get_order();
+        }
```

Comment:
This property doesn't add much value IMO
In any case, if needed, maybe it's better to add it in the `pos_order_to_sale_order` module instead